src/jenesis/contracts/detect.py

```diff
@@ -57,6 +57,10 @@ def parse_contract(name: str) -> Contract:
             cargo_root = os.path.join(contracts_folder, name)
         artifacts_path = os.path.join(cargo_root, 'artifacts')
         binary_path = os.path.abspath(os.path.join(artifacts_path, f'{contract_name}.wasm'))
+        
+        # replace hyphens with underscores to match rust naming convention
```

Comment:
We can just replace the hyphens here since that's what rust does with the actual binary path.